src/openrct2/world/map.h

```diff
@@ -260,25 +260,29 @@ enum
 };
 
 enum {
-    TILE_ELEMENT_SLOPE_NE_CORNER_UP = 1,
-    TILE_ELEMENT_SLOPE_SE_CORNER_UP = 2,
-    TILE_ELEMENT_SLOPE_SW_CORNER_UP = 4,
-    TILE_ELEMENT_SLOPE_NW_CORNER_UP = 8,
+    TILE_ELEMENT_SLOPE_FLAT = 0,
 
-    TILE_ELEMENT_SLOPE_NW_CORNER_DN = 7,
-    TILE_ELEMENT_SLOPE_SW_CORNER_DN = 9,
-    TILE_ELEMENT_SLOPE_SE_CORNER_DN = 13,
-    TILE_ELEMENT_SLOPE_NE_CORNER_DN = 14,
+    TILE_ELEMENT_SLOPE_ALL_CORNERS_UP = 15,
 
-    TILE_ELEMENT_SLOPE_E_SIDE_UP = 3,
-    TILE_ELEMENT_SLOPE_S_SIDE_UP = 6,
-    TILE_ELEMENT_SLOPE_N_SIDE_UP = 9,
-    TILE_ELEMENT_SLOPE_W_SIDE_UP = 12,
+    TILE_ELEMENT_SLOPE_NE_CORNER_UP = (1 << 0),   // 0b0001
+    TILE_ELEMENT_SLOPE_SE_CORNER_UP = (1 << 1),   // 0b0010
+    TILE_ELEMENT_SLOPE_SW_CORNER_UP = (1 << 2),   // 0b0100
+    TILE_ELEMENT_SLOPE_NW_CORNER_UP = (1 << 3),   // 0b1000
 
-    TILE_ELEMENT_SLOPE_NW_SE_VALLEY = 5,
-    TILE_ELEMENT_SLOPE_NE_SW_VALLEY = 10,
+    TILE_ELEMENT_SLOPE_NW_CORNER_DN = 7,   // 0b0111
```

Comment:
@duncanspumpkin Do you think it's better to rewrite all the following enum in a more clean way, eg. `TILE_ELEMENT_SLOPE_NW_CORNER_DN = TILE_ELEMENT_SLOPE_ALL_CORNERS_UP & ~TILE_ELEMENT_SLOPE_NW_CORNER_UP` and/or `TILE_ELEMENT_SLOPE_E_SIDE_UP = TILE_ELEMENT_SLOPE_NE_CORNER_UP | TILE_ELEMENT_SLOPE_SE_CORNER_UP`?